packages/http-exception-filter/README.md

```diff
@@ -0,0 +1,135 @@
+<p align="center">
+  <a href="http://nestjs.com"><img src="https://nestjs.com/img/logo_text.svg" alt="Nest Logo" width="320" /></a>
+</p>
+
+<p align="center">
+  A <a href="https://github.com/nestjs/nest">Nest</a> http exception filter.
+</p>
+
+# NestJS Http Exception Filter
+
+A simple NestJS Http [Exception Filter](https://docs.nestjs.com/exception-filters).
+
+## Installation
+
+```bash
+npm install --save @algoan/http-exception-filter
+```
+
+## Usage
+### Default usage
+Use the interceptor as a global interceptor (cf. refer to the [last paragraph](https://docs.nestjs.com/interceptors#binding-interceptors) of this section for more details).
```

Comment:
Actually, it isn't an interceptor, it is an [exception filter](https://docs.nestjs.com/exception-filters).